tests/playwright/src/model/pages/create-kind-cluster-page.ts

```diff
@@ -0,0 +1,48 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { expect as playExpect, type Locator, type Page } from '@playwright/test';
+
+import { BasePage } from './base-page';
+
+export class CreateKindClusterPage extends BasePage {
+  readonly clusterPropertiesInformation: Locator;
+  readonly clusterNameField: Locator;
+  readonly clusterIngressControllerCheckbox: Locator;
+  readonly clusterCreationButton: Locator;
+  readonly goBackButton: Locator;
+  readonly logsButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.clusterPropertiesInformation = this.page.getByRole('form', { name: 'Properties Information' });
+    this.clusterNameField = this.clusterPropertiesInformation.getByLabel('Name');
+    this.clusterIngressControllerCheckbox = this.clusterPropertiesInformation.getByRole('checkbox');
```

Comment:
please add accessible name into getByrole. And, please add a locator for a parent element, that would allow you to actually change the checkbox value. Known issue. This is a workaround